dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -86,3 +89,12 @@ includePathInWASReqURL.desc=Setting the Path parameter can allow the client/brow
 
 trackLoggedOutSSOCookies=Track logged out LTPA single signon tokens
 trackLoggedOutSSOCookies.desc=Specifies whether to track LTPA single signon tokens that are logged out on a server so that it can not be reused on the same server.
+
+overrideHttpAuthenticationMechanism=Name of HttpAuthenticationMechanism to override
+overrideHttpAuthenticationMechanism.desc=Specifies the HttpAuthenticationMechanism for overriding an application-defined HttpAuthenticationMechanism. The HttpAuthenticationMechanism is defined by the Java EE 8 Security specification. The acceptable value is BASIC or FORM. When FORM is used, specify the loginFormURL and loginErrorURL properties.
```

Comment:
Minor edit:
Sorry I didn't catch this before, but could we please change "The HttpAuthenticationMechanism is defined by the Java EE 8 Security specification." to "The Java EE 8 Security specification defines the HttpAuthenticationMechanism." ?

Reply:
Got it. I'll change it as suggested.